packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -456,7 +454,7 @@ class Uploader {
    * @param {string} url
    * @param {object} extraData
    */
-  emitSuccess (url, extraData) {
+  _emitSuccess (url, extraData) {
```

Comment:
Should we work on replacing all leading underscores by `#` (so private fields and private methods) now that we have dropped support for Node.js 12?